### ISSUE TITLE
Packaged sed utility dumped core

### ISSUE THREAD

**Issue by amitavmohanty01** [ValveSoftware/Dota-2#1743]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): https://pastebin.fun/k5AQC7Lnre
* Have you checked for system updates?: [Yes/No] Yes
* Are you using the latest stable video driver available for your system? [Yes/No] Yes
* Have you verified the game files?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
I was playing Dota 2 and the game crashed. I had to restart.

It seems the dump is from the `sed` utility which comes packaged.

```
           PID: 7441 (sed)
           UID: 1000 (dknight)
           GID: 100 (users)
        Signal: 11 (SEGV)
     Timestamp: Sat 2020-05-09 23:26:43 IST (2 days ago)
  Command Line: sed -e s|-|/|
    Executable: /usr/bin/sed
 Control Group: /user.slice/user-1000.slice/session-2.scope
          Unit: session-2.scope
         Slice: user-1000.slice
       Session: 2
     Owner UID: 1000 (dknight)
       Boot ID: 2bcedbf87119489f82be3cf311e96d83
    Machine ID: f52d0eeb525f4cd584fe54d97e927133
      Hostname: clown.man.machine
       Storage: /var/lib/systemd/coredump/core.sed.1000.2bcedbf87119489f82be3cf311e96d83.7441.1589047003000000000000.lz4
       Message: Process 7441 (sed) of user 1000 dumped core.
                
                Stack trace of thread 7441:
                #0  0x000060e142d25575 n/a (/home/dknight/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so + 0xa575)

```

I am attaching the dump as well.
[sed.zip](https://github.com/ValveSoftware/Dota-2/files/4616434/sed.zip)

**Comment by kisak-valve** [comment]
Hello @amitavmohanty01, /usr/bin/sed is a distro-installed utility and not bundled with Dota 2. While this may indicate an issue elsewhere, using 3 issue reports to report the same incident is excessive. (#1742, https://github.com/ValveSoftware/steam-for-linux/issues/7105, and here) Feel free to add a note to #1742.

**Comment by amitavmohanty01** [comment]
@kisak-valve it might appear that I am spamming but I created three reports because I thought they were different issues. At least from the core dumps, I found one to be related to `sed`, another to `steam` and another to `dota`. Do we know that they are related ? As I said in https://github.com/ValveSoftware/Dota-2/issues/1742, I have a very large dump that I want to share but do not know the proper channel.
